apps/web/modules/settings/platform/managed-users/managed-users-view.tsx

```diff
@@ -0,0 +1,95 @@
+"use client";
+
+import { useState } from "react";
+
+import Shell from "@calcom/features/shell/Shell";
+import { PlatformManagedUsersTable } from "@calcom/features/users/components/UserTable/PlatformManagedUsersTable";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import type { PlatformOAuthClient } from "@calcom/prisma/client";
+import { SkeletonContainer, SkeletonText } from "@calcom/ui";
+
+import { useOAuthClients } from "@lib/hooks/settings/platform/oauth-clients/useOAuthClients";
+
+import { OAuthClientsDropdown } from "@components/settings/platform/dashboard/oauth-client-dropdown";
+
+export const ManagedUsersSkeletonLoader = () => {
```

Comment:
If you create it as function
```
export function ManagedUsersSkeletonLoader() {...}
```
then you can move it under `ManagedUsersView` to make code more readable - higher order functions are on the top so that we can read code top to bottom.